src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.jsx

```diff
@@ -213,96 +33,44 @@ const CustomFieldInputMultipleChoice = forwardRef(function CustomFieldInputMulti
         });
     };
 
-    getChoices();
+    fetchChoices();
   }, []);
 
+  useEffect(() => {
+    setFullPropValue(props.value.map((choiceID) => {
+      return allChoices.find(choice => choice.id === choiceID);
+    }).filter(mappedChoice => mappedChoice !== undefined));
```

Comment:
When is a mapped choice undefined? This might be indicative of a missing state for the component -- e.g. "if we haven't fully loaded the selected choices then we need to continue to be in a loading state". 

But that could be an iteration step... First, I would like to understand how it's `undefined`.

Reply:
Actually, this was intended to protect against choices saved in something like a custom form that no longer exist on the backend. The loading state is (kind of) already covered by the `MultiAutocompleter` which doesn't currently expose a `loading` prop. The assumption is that loading of an unfiltered set of choices will return more quickly than the filtered set the `MultiAutocompleter` loads, but that may turn out to be faulty.

I think the potential iteration step here is more along the lines of "allow `props.loading` to be set in such a way that the internal loading state is used _after_ the `props.loading` state becomes false".